src/client/actions/dataflows.js

```diff
@@ -10,7 +10,14 @@ const dataflowsLoaded = dataflows => ({
 });
 
 export const search = value => dispatch => {
-  requestJson({ dispatch, method: 'post', url: '/api/search', body: { search: value }, message: 'Cannot search dataflows', status: 'DANGER' })
+  requestJson({
+    dispatch,
+    method: 'post',
+    url: '/api/search',
+    body: { search: value },
+    header: 'Runtime error',
+    message: 'Cannot search dataflows',
+    status: 'DANGER',
```

Comment:
NO, status is hardcoded inside manageError from requestJson, Runtime error is a default value, only pass 'Cannot ...'